Demo/Demo/Models/Order.swift

```diff
@@ -1,4 +1,4 @@
-struct Order: Codable {
+struct Order: Codable, Equatable {
```

Comment:
Why do these need to conform to Equatable?

Reply:
Because the I observe changes in the state to scroll to bottom of view when new result view is shown.